docs/resources/subaccount_environment_instance.md

```diff
@@ -53,8 +52,8 @@ resource "btp_subaccount_environment_instance" "cloudfoundry" {
   environment_type = "cloudfoundry"
   service_name     = "cloudfoundry"
   plan_name        = "standard"
-  # some regions offer multiple environments of a kind and you must explicitly select the target environment in which
-  # the instance shall be created. 
+  # ATTENTION: some regions offer multiple environments of a kind and you must explicitly select the target environment in which
+  # the instance shall be created unsing the parameter landscape label. 
```

Comment:
typo.

Reply:
Also do we need to add a pointer to [regions](https://help.sap.com/docs/btp/sap-business-technology-platform/regions) when talking about multiple environment?

Reply:
Do we mention the extension landscapes (e.g cf-eu10-002) also here.

Reply:
@vipinvkmenon Typo is fixed. 
Region is a topic of the subaccount resource. You cannot influence the region on environment instance level (except for cluster regions which is specific to Kyma and part of the parameter set there that we referenced in the example

Reply:
Makes Sense 👍